rpc/gnmi/gnmi-union_replace.md

```diff
@@ -494,16 +494,24 @@ union_replace: {
 ## 7. gNSI, bootz and gNMI interoperation
 
 Configuration affected by a gNMI `SetRequest` is defined to be Dynamic
-Configuration.  THe term ‘Dynamic’ is used because this configuration is
+Configuration.  The term ‘Dynamic’ is used because this configuration is
 expected to be modified while the NOS is running.   Security configuration is a
 special case of dynamic configuration and is managed by gNSI.  Configuration
 necessary to boot and make a factory default NOS manageable by gNMI is defined
 as bootz configuration.
 
-bootz configuration items can be expressed as OC, NY and CLI. Bootz defines an
-order (or precedence) to apply the configuration name spaces to the NOS.  The
-order is dynamic configuration, followed by bootz configuration, followed by
-gNSI configuration.
+bootz [boot_config](https://github.com/openconfig/bootz/blob/e0eb604d8e7e089ecf0d4553cce79de769328173/server/entitymanager/proto/entity.proto#L55)
```

Comment:
the references should point to the bootz.proto https://github.com/openconfig/bootz/blob/e0eb604d8e7e089ecf0d4553cce79de769328173/proto/bootz.proto

Reply:
Nice catch, fixed.